src/server/lib/renderer.ts

```diff
@@ -23,12 +25,20 @@ interface RendererOpts {
 
 const { gaUID } = getConfiguration();
 
-export const renderer: (url: string, opts?: RendererOpts) => string = (
-  url,
-  opts = {},
-) => {
+export const renderer: (
+  url: string,
+  request: Request,
+  opts?: RendererOpts,
+) => string = (url, request, opts = {}) => {
   const { globalState = {}, queryParams = {}, pageTitle = 'Gateway' } = opts;
 
+  try {
```

Comment:
set browserName on the server to avoid adding bowser as a clientside dependency
(it would add 25KB to the bundle)